src/main/resources/templates/client/cypher/list.html

```diff
@@ -16,17 +17,17 @@ <h1 th:utext="#{label.cyphers}">Šifry</h1>
                             <div th:text="|${cypherGameInfo.getStage()}. ${cypherGameInfo.getName()}|" class="col"></div>
                             <div th:unless="${cypherGameInfo.getCount() == 0}" class="col hints">
                                 <span class="count" th:text="${cypherGameInfo.getCount()} + ' ×'"></span>
-                                <i class="far fa-lightbulb"></i>
+                                <em class="far fa-lightbulb"></em>
```

Comment:
Proc si nahrazoval tag?

Reply:
sonar to nahlasil
![image](https://user-images.githubusercontent.com/47707682/66839931-7f6c8880-ef67-11e9-95da-fe8835770a73.png)

Reply:
Hmmm, sonar bych pouzival asi jen na Javu.
Co tam mas vsechno zaple za pravidla ty jo? To my tam vubec nemame.

Reply:
mam nejaky plugin a ten mi to nahlasil :O pravidla jsem nezkoumal

Reply:
vratil jsem tam zpatky i tag

Reply:
Já jen, že -i- tag je použitý i v dokumentaci font-awesome knihovny pro ty ikonky.